calculator_simple/build/macos/Makefile

```diff
@@ -0,0 +1,37 @@
+# GNU Make 4.3
+# macOS Build Script
+#
+# Tommy Vegetables (leafyrabbet)
+# leafyrabbet@gmail.com
+# Copyright (c) 2020, TenonGarden Productions
+
+.RECIPEPREFIX = +
+
+CC_STD = c++2a
+GCC := gcc-9 --std=$(CC_STD)
+FLAGS = -Wall -pedantic -g
```

Comment:
I think `-Werror` and `-Wextra` should be added back in here, and maybe change `-g` to `-ggdb`, but make it configurable through a CLI flag like `--DEBUG` or something?

Reply:
Partially resolved in 7427670 -- `-ggdb` not (yet) added.

Reply:
New variable `FLG_GDB_DEBUG` added in fb2b286, resolving this request.